pyproject.toml

```diff
@@ -119,6 +120,7 @@ per-file-ignores = [
   "mpt_api_client/resources/catalog/mixins.py: WPS110 WPS202 WPS214 WPS215 WPS235",
   "mpt_api_client/resources/catalog/products.py: WPS204 WPS214 WPS215 WPS235",
   "mpt_api_client/rql/query_builder.py: WPS110 WPS115 WPS210 WPS214",
+  "mpt_api_client/utils.py: WPS100 WPS453",
```

Comment:
:( ... why are we broken the WPS100 rule if we are creating new things?

<img width="697" height="460" alt="Image" src="https://github.com/user-attachments/assets/42a80aec-238c-41b9-ab70-c52ee74ccf2c" />